sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -20,14 +20,23 @@ async function main(): Promise<void> {
 
   console.log("Creating and sending a batch of events...");
   try {
-    const batch = await producer.createBatch();
+    // create a batch targeted to a particular partition
+    const batch = await producer.createBatch({
+      partitionId: "0"
+    });
+
+    // add events to our batch
     for (let index = 0; index < 10; index++) {
+      // messages can fail to be added to the batch if they exceed the maximum size configured for
+      // the EventHub.
```

Comment:
In the review meeting, there were talks about updating the send sample to show a more "streaming" case of incoming data that need to be sent.

Do we plan to make that change in a separate PR and track in a separate issue?

Reply:
Issue created: #6254